addons/stock/views/stock_quant_views.xml

```diff
@@ -257,6 +257,7 @@
                 <field name="name" string="Package Name"/>
                 <field name="location_id"/>
                 <field name="package_type_id"/>
+                <filter string="In internal locations" name="internal" domain="['|', ('location_id.usage', '=', 'internal'), ('location_id', '=', False)]"/>
```

Comment:
Should this group be available even without `Storage Locations` enabled in the Settings?
The same could be said about the `Location` filter a few lines under this one.